MetabolicAtlas/MetabolicAtlas
# feat: refactor citations info

**Related issue(s) and PR(s)**  
This PR closes #1153.  Started by @e0 .


**Type of change**  

 
**List of changes made**  

- Home page, GotEnzymes and the About-Citations page all use the same source of information for publications.


**Testing**  

- Visit `/about/platform#citation` and make sure the citation info looks as is should
- Visit `/gotenzymes` and make sure the citation info looks as is should
- Visit `/` (the start page) and make sure citation info looks as is should


**Further comments**  

There is a new colon added to the Version 3 publication. To me that is acceptable, and also makes sense  when compared to the other publications.
New:
![2022-12-09-154644_546x64_scrot](https://user-images.githubusercontent.com/1029190/206727962-8c430908-fa9c-480b-9318-2af1b19f7894.png)
Old:
![2022-12-09-154658_599x61_scrot](https://user-images.githubusercontent.com/1029190/206728095-f24fc11a-1309-4a1c-83b7-ea54e62e8423.png)



**Definition of Done checklist**
